### ISSUE TITLE
Lymph node abscess

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6067]
**Preferred term label:**
 Lymph node abscess

**Synonyms**
Lymphatic node abscess

**Definition (free text, please give PubMed ID)**
A collection of pus in the lymph nodes.
PMID: 31395094

**Parent term (use hpo.jax.org/app)**
Abscess

**Comment by pnrobinson** [comment]
added
